resources/crds/hammerdb_crds/mariadb/ripsaw_v1alpha1_hammerdb_mariadb_server.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: mariadb-db
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mariadb-custom-config
+  namespace: mariadb-db
+data:
+  custom.conf: |
+        # This group is read both both by the client and the server
+        # use it for options that affect everything
+        #
+        [client-server]
+        #
+        # include all files from the config directory
+        #
+        !includedir /etc/my.cnf.d
+        port=3306
+        ssl=0
+        [mysqld]
+        innodb_file_per_table
```

Comment:
Missing the value for this config option, should be `ON` or `OFF`

Reply:
Nice catch, change it innodb_file_per_table=1